tests/acceptance/features/apiOcm/share.feature

```diff
@@ -1229,3 +1229,25 @@ Feature: an user shares resources using ScienceMesh application
         }
       }
       """
+
+  @issue-10213
+  Scenario: local user deletes access of shared resource to federated user
+    Given using spaces DAV path
+    And using server "REMOTE"
+    And "Brian" has created the federation share invitation
+    And using server "LOCAL"
+    And "Alice" has accepted invitation
+    And user "Alice" has created a folder "FOLDER" in space "Personal"
+    And user "Alice" has sent the following resource share invitation to federated user:
+      | resource        | FOLDER   |
+      | space           | Personal |
+      | sharee          | Brian    |
+      | shareType       | user     |
+      | permissionsRole | Editor   |
+    When user "Alice" removes the access of user "Brian" from resource "FOLDER" of space "Personal" using the Graph API
+    Then the HTTP status code should be "204"
+    When using server "REMOTE"
+    And user "Brian" lists the shares shared with him without retry using the Graph API
+    Then the HTTP status code should be "200"
+#    And user "Brian" should have a share "FOLDER" shared by user "Alice" from space "Personal"
```

Comment:
please open up reason behind commenting step look @issue-10213 has been closed so this issue can't be reason

Reply:
My bad! Forgot to remove the commented step.